remo/events/api/views.py

```diff
@@ -19,11 +19,23 @@
 KPI_WEEKS = 12
 
 
+class EventsFilter(django_filters.FilterSet):
+    owner = django_filters.CharFilter(name='owner__userprofile')
+    categories = django_filters.CharFilter(name='categories__name')
+    initiative = django_filters.CharFilter(name='campaign__name')
+
+    class Meta:
+        model = Event
+        fields = ('name', 'description', 'start', 'end', 'city', 'region',
```

Comment:
I would suggest to remove completely `start` and `end` and enable them in another PR which will include 'gt', 'lt' etc filters. Moreover it makes more sense to filter against date and not datetime in events. Thoughts?